qemu/tests/cfg/expose_host_mtu.cfg

```diff
@@ -0,0 +1,16 @@
+- expose_host_mtu: install setup image_copy unattended_install.cdrom
+    no RHEL.6 RHEL.5 RHEL.4 RHEL.3
+    no Windows
+    requires_root = yes
+    type = expose_host_mtu
+    kill_vm = yes
+    only virtio_net
+    start_vm = no
+    nic_extra_params = ",host_mtu=4000"
```

Comment:
I would suggest take `4000` as a variable, would this be better?
```
mtu_value = 4000
nic_extra_params = ",host_mtu=${mtu_value}"
```
```
set_guest_mtu_cmd = "ifconfig %s mtu ${mtu_value}"
```
L#66: `if "mtu %s" % params["mtu_value"] in output:`
So does L#67 and L#75.

Reply:
@vivianQizhu Updated. Thanks.